### DISCUSSION TITLE
ng zorro pro version

### DISCUSSION THREAD

**Discussion by AgustinLaValla** [NG-ZORRO/ng-zorro-antd#6552] [Q&A] [answered]
Hi everyone. I have a question. Is there a pro version of ant design for Angular like this one (that's for React)?: 
  https://v2-pro.ant.design

  I would like to the implement these components in my project. Thank you.

**Comment by vthinkxie** [comment, marked as answer]
you can try https://github.com/ng-alain/ng-alain
